test/api/projects/instance-get/004-get-ok/expected-response.json

```diff
@@ -1,6 +1,9 @@
 {
   "versionDetail": {
-    "components": [],
+    "components": [
+      "foo.bar",
+      "keboola.wr-snow%s"
+    ],
```

Comment:
A API vrati konkretne `componentId` pre Snowflake Writer, pre dany stack.
Je to vzdy ta ista komponenta (kod) ale ma ine `componentId`, preto ze ma iny `staging-storage`.

Reply:
(zmeny v sablonach budu vyzerat takto: https://github.com/keboola/keboola-as-code-templates/pull/127)